# publish_rate in Hz
publish_rate: 1000.0
# Possible units: [mm, cm, m]
units: mm
# Workspace [width, depth, height] = [x, y, z]
hysteresis: 3.0
workspace:
  width: 100.0
  depth: 100.0
  height: 100.0
  center: [0, 0, 0]
# Position parameters
reference_frame: world
position_ratio: 250
pivot_dist: 5.0
axes_mapping: [-y, x, z]
# Force feedback parameters
k_center: 0.15
b_center: 0.003
k_rate: 0.01
b_rate: 0.003
rate_gain: 0.05
vibration:
  a: 1.0
  c: 5.0
  frequency: 30.0
  duration: 0.3
# Rotation between master and slave end efector
axes_rotation_1: [1, -0, 0]
angle_rotation_1: 3.14
#~ angle_rotation_1: 1.570796
axes_rotation_2: [0, -1, 0]
angle_rotation_2: 1.570796
axes_rotation_3: [-1, 0, 0]
angle_rotation_3: 0.0
